image/templates/helm/shared/templates/02-scanner-06-deployment.yaml.htpl

```diff
@@ -49,6 +49,31 @@ spec:
       {{- if ._rox.scanner.priorityClassName }}
       priorityClassName: {{ ._rox.scanner.priorityClassName }}
       {{- end }}
+      {{- if ._rox.env.securedClusterCertRefresh }}
+      initContainers:
+        - name: init-tls-certs
+          image: {{ quote ._rox.image.main.fullRef }}
+          command:
+          - /stackrox/bin/init-tls-certs
+          args:
+          - --legacy=/run/secrets/stackrox.io/certs-legacy/
```

Comment:
There's an inconsistency here:

In the manifests we use double-dash syntax, but OTOH:

```
❯ ./init-tls-certs --help
Usage of ./init-tls-certs:
  -destination string
        Destination directory for certs
  -legacy string
        Source directory for legacy certs
  -new string
        Source directory for new certs
```

Reply:
The `flags` package supports both `-` and `--` for arguments (but the default is `-`), I used `--` because it's conventionally used for long parameters.

Reply:
Hm, alright!